clouddriver-ecs/src/main/java/com/netflix/spinnaker/clouddriver/ecs/cache/Keys.java

```diff
@@ -61,6 +61,10 @@ public String getCloudProvider() {
 
   @Override
   public Boolean canParseType(String type) {
+    return canParse(type);
+  }
+
+  private static Boolean canParse(String type){
```

Comment:
Please make best efforts to follow the google coding style guide (in the case of this PR, adequate whitespace between conditional expressions, parens, braces).

Reply:
Done.